docs/architecture/api_draft/compute_pilot_description.py

```diff
@@ -6,12 +6,12 @@
 # ------------------------------------------------------------------------------
 #
 
-class ComputePilotDescription (Description) :
```

Comment:
Because this also renames `ComputePilot` to `Pilot`.  We don't have any other pilot.  Back in the day we planned for `DataPilot`s and `DataUnit`s - but thise will not happen.  We may have a data staging service at some point (and our stager components move in that direction), but no separate pilot job.

Reply:
Sorry, if it was not clear. I did not mean the class name, but the file name. It's still `compute_pilot_description.py`. Will it change to `pilot_descirption.py`?

Reply:
Thanks, done.